src/data/disks.rs

```diff
@@ -145,8 +163,47 @@ extern "C" {
 
 #[cfg(not(windows))]
 fn load_disks(canonicalize_paths: bool) -> Vec<Disk> {
-    sysinfo::Disks::new_with_refreshed_list()
-        .iter()
-        .map(|d| Disk::from_sysinfo_disk(d, canonicalize_paths))
-        .collect()
+
+    let mut result: Vec<Disk> = Vec::new();
+    let mut seen_mount_points = std::collections::HashSet::new();
+
+    // Use sysinfo to get system disks
+    let disks = sysinfo::Disks::new_with_refreshed_list();
+    for disk in &disks {
+        let mount_point = disk.mount_point();
+        if mount_point == Path::new("/") {
+            // Skip the root system drive
+            continue;
+        }
```

Comment:
At least on Linux, it is still important that the root volume is listed. However, feel free to display the disk name as `root`.

Reply:
On macOS the root is already displayed as `Macintosh HD`